### DISCUSSION TITLE
您好！请问在源码中哪里增加随机暂停时间。

### DISCUSSION THREAD

**Discussion by GFeelingh** [VermiIIi0n/fuckZHS#64] [Q&A] [answered]
作者您好！请问在源码中哪里增加随机暂停时间，现在过一会还是会检测让输入验证码！感谢作者的奉献！

**Comment by VermiIIi0n** [comment, marked as answer]
[这儿嘞](https://github.com/VermiIIi0n/fuckZHS/blob/e48e1c677ef3d43d5f4a031d57e2c2828f84383b/fucker.py#L472)

**Reply by GFeelingh** [reply]
十分感谢！！！！爱您😍
